data/human/human missions.txt

```diff
@@ -7455,3 +7455,1836 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+
+
+mission "Timothy Radrickson 1: Stowaway"
+	invisible
+	landing
+	source
+		near "Zeta Aquilae" 3 10
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
+	destination "Rand"
+	passengers 1
+	to offer
+		random < 10
+	on enter
+
+	# You will see this conversation after the first jump you make.
+	on enter
+		conversation
+			`Upon completing your jump, one of your internal sensors begins to beep. You flick off the siren and check the diagnostic report: apparently there is a non-negligible heat source detected in the food storage of your ship.`
+			`	As problems go, it is not a particularly alarming one: there are plenty of reasons there may be a heat spike in storage, and most of them would not require the captain of the ship to investigate.`
+			`	It is unusual, however.`
+			choice
+				`	(Investigate.)`
+					goto check
+				`	(Ignore the sensor.)`
+
+			action
+				fail
```

Comment:
I believe the fail is necessary because `Timothy Radrickson 2: mummy` only fires if you fail this mission (and I placed the failures in places where Timothy would die to your inaction). Moreover it immediately removes the mission from your active missions list while `decline` doesn't actually do anything but end the conversation in this instance. (You have technically already accepted `Timothy Radrickson 1: Stowaway` when you see this conversation, it's not an `on offer` conversation.)

I had thought in the past to streamline it and simply remove that second option, but I have no time to at the moment.
```
mission "Timothy Radrickson 2: mummy"
[...]
	to offer
		has "Timothy Radrickson 1: Stowaway: failed"
```